tests/h/accounts/schemas_test.py

```diff
@@ -126,6 +126,36 @@ def test_it_is_invalid_with_invalid_characters_in_username(self,
                                                   "numbers, periods, and "
                                                   "underscores.")
 
+    def test_it_is_invalid_with_false_privacy_accepted(self,
+                                                            pyramid_request):
+        schema = schemas.RegisterSchema().bind(request=pyramid_request)
+
+        with pytest.raises(colander.Invalid) as exc:
+            schema.deserialize({"privacy_accepted": False})
```

Comment:
This test passes, but the actual value that `deform` passes to `deserialize` comes from the `deform.CheckboxWidget`'s `false_val` constructor argument. This is literally the string `"false"`. I only know this because I read the deform code, it isn't very clear otherwise.